### ISSUE TITLE
Support for multiple bids per bidder in stored requests

### ISSUE THREAD

**Issue by jarrettabello** [prebid/prebid-server#2459]
I have been looking high and low for a way to support multiple bid parameters per bidder in a stored request. Given that an openRTB stored request has support for multiple sizes per "unit" -  I was expecting to find a way to configure multiple bids per bidder. 

For instance in Prebid.js you can configure a ad unit to have multiple sizes i.e. a medium rectangle that allows 300x250s and 300x600s. Depending on the SSP one might be required to request a unique bid for each size. in PBJS this is not a problem because bids are an array. However, in Prebid Server and the OpenRTB standard, from what I can find, there appears only to be support for a single bid configuration per bidder per stored request given that bidder is an **unique key.** 

Before I go and try to incorrectly use bidder aliases to achieve this I was wondering if anyone had run across this limitation. If there is a way to do this currently I would love some guidance. 

Thank you!

**Comment by jarrettabello** [comment]
I appreciate the response. Maybe I am thinking about this the wrong way having lived in PBJS world for so long. This connection we are creating is from the Mobile SDK to PBSGO using stored imps on the server side. I am trying to work through the logic to attempt to match our PBJS setup. 

Given the above, would you mind explaining at a high level how the server-side adapter would be responsible for handing the unique bids per size. Would the APP have to send two requests for a single unit? Im just attempting to future proof so I don't have a specific bidder,  but obviously agree that single bid per multi size is the optimal solution. (even on web)

**Comment by bretg** [comment]
> how the server-side adapter would be responsible for handing the unique bids per size. Would the APP have to send two requests for a single unit? 

right. there's precedence for server-side adapters that split up the OpenRTB and make multiple calls to their backend, whether one-per-size or one-per-mediatype.
